### PR TITLE
chore:  Added Sanity suite Page for data residency

### PR DESCRIPTION
**Pull request by rspriyamdixit**
## PR Description
1. Added Sanity suite Page for data residency
2. Page does not have load api.
3. This is done to support send different load api parameters at run time to automate data residency TC.
Please include a summary of the change along with the relevant motivation and context.

## Notion ticket

Ticket link

## Screenshots

Please add screenshots for any new features or UI bug fixes for the following browsers -

- Chrome
  >
- Firefox
  >
- Safari
  >

## Security

- [ ] The code changed/added as part of this pull request won't create any security issues with how the software is being used.

### CODE REVIEW

File: packages/sanity-suite/public/v3/dataResidency/index-cdn.html [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+          oneTrust: {
+            enabled: true,
+          },
+        },
+        storage: {
+          migrate: true
+        }
+      };
+
+      // rudderanalytics.load(
```
Comment (bardisg) [inline packages/sanity-suite/public/v3/dataResidency/index-cdn.html:96]: why is this commented out?

**Comment by bardisg** [comment]
Can we add the v1.1 folders too?

File: packages/sanity-suite/public/v3/dataResidency/index-cdn.html (hunk 1 above)
Comment (rspriyamdixit) [reply packages/sanity-suite/public/v3/dataResidency/index-cdn.html:96]: We call ` load api` at runtime with different `loadoptions` to test all Data Residency test cases.

**Comment by rspriyamdixit** [comment]
> 



> Can we add the v1.1 folders too?

Sure,  will create a PR with v1.1 test cases as well. And create task in linear to track it.

**Review by bardisg — APPROVED** [review]

**Pull request merged**